packages/documentation/stories/components/alert/alert.stories.tsx

```diff
@@ -0,0 +1,238 @@
+import React from 'react';
+import parse from 'html-react-parser';
+import { useArgs } from '@storybook/client-api';
+import docsPage from './alert.docs.mdx';
+
+import '@swisspost/design-system-styles/basics.scss';
+import '@swisspost/design-system-styles/components/alert.scss';
+import '@swisspost/design-system-styles/components/icons.scss';
+import './alert.stories.scss';
+
+export default {
+  title: 'Components/Alert',
+  parameters: {
+    docs: {
+      page: docsPage
+    },
+    controls: {
+      exclude: [
+        'Icon',
+        'Icon Color',
+        'No Icon',
+        'Dismissible',
+        'Fixed'
+      ]
+    }
+  },
+  args: {
+    title: 'Titulum',
+    content: '<p>Contentus momentus vero siteos et accusam iretea et justo.</p>',
+    variant: 'alert-primary',
+    icon: '',
+    iconVariant: '',
+    noIcon: false,
+    dismissible: false,
+    dismiss: false,
+    fixed: false,
+    show: false
+  },
+  argTypes: {
+    title: {
+      name: 'Title',
+      control: { type: 'text' },
+      table: { category: 'Content' }
+    },
+    content: {
+      name: 'Content',
+      control: { type: 'text' },
+      table: { category: 'Content' }
+    },
+    variant: {
+      name: 'Variant',
+      control: {
+        type: 'select',
+        labels: {
+          'alert-primary': 'Primary',
+          'alert-notification': 'Notification',
+          'alert-success': 'Success',
+          'alert-danger': 'Danger',
+          'alert-error': 'Error',
+          'alert-warning': 'Warning',
+          'alert-info': 'Info'
+        }
+      },
+      options: [
+        'alert-primary',
+        'alert-notification',
+        'alert-success',
+        'alert-danger',
+        'alert-error',
```

Comment:
Should we deprecate one of the error/danger classes as well?  Bootstrap does not provide error, only danger.

Reply:
In my opinion we could remove the `alert-error` variant.

Reply:
#825